### DISCUSSION TITLE
申请退款冷处理，但极速把我账号变免费版了？？

### DISCUSSION THREAD

**Discussion by Ginny125** [getlantern/lantern-forum-cn#1070] [General]
七天前申请退款发了账号、支付记录、问题报告截图，清清楚楚明明白白地，结果只是把我账号从专业版变免费版？？后台改账号权限这么简单怎么退款就磨磨唧唧，欸凭什么你们拿着钱还把我专业版账号给撸了？没退款就把我账号变回去。
核心诉求还是要求退款，但退款前我的账号必须恢复成专业版，不管你们退全款还是按使用天数折算
账号邮箱[wawzj1255173656@163.com]
问题报告[79565012]
![微信图片_20241125121321](https://github.com/user-attachments/assets/c10e72ef-d55a-457d-83f6-e311ce1f417c)
![微信图片_20241125121707](https://github.com/user-attachments/assets/3bc54f98-97c1-462d-8d21-fe1698f07aba)
![微信图片_20241205140642](https://github.com/user-attachments/assets/eafd2972-cd6e-4ff0-8d8b-e8396b069661)

**Comment by Ginny125** [comment]
@Mike777Khoo 还有，交易平台申诉退款没用，显示商家已处理，但就是不退款，我现在要求你们支付部门退款，技术部门在我收到退款以前把我免费版的账号还原成专业版

**Reply by alexa1111ee** [reply]
。。。。我也是。。。。一样的申请退款 一样的变成了免费版本。。找不到升级版的入口。。真是服了，。简直了

**Comment by MaxMKMax** [comment]
@Ginny125 您好. 我们没有把您的账号变免费. 是系统出现问题吧, 或者您登出了专业版吗? 
或者就是退款已经在处理. 系统就自动处理了. 退款完成的过程需要 2 周. 谢谢您的耐心等待!

**Reply by Ginny125** [reply]
@Mike777Khoo 确认过不是我的问题导致专业版登出，而是移动端和pc端都被登出，2周最好不是你们的托辞
![Uploading image.png…]()

**Reply by Ginny125** [reply]
![Uploading Snipaste_2024-12-06_09-47-37.png…]()

**Reply by liuh23** [reply]
我的也是，已经快两个月了

**Comment by Ginny125** [comment]
@Mike777Khoo 你好，收到邮件提示我{79565012}该ticket已关闭，但我并未收到退款，请问退款流程到哪一步了？还有，麻烦根据上面的付款截图告知我退款方式，支付宝方申请退款已经尝试过不可用。
